snakePipes/shared/rscripts/CSAW.R

```diff
@@ -58,13 +58,14 @@ pe_param <- csaw::readParam(max.frag = 500, pe = pe)  # Some CSAW functions expl
 ## Read data
 ##filter out input using yaml
 library(yaml)
-y<-read_yaml(yaml_path)
-input_list<-unique(unlist(lapply(y[[1]],function(X)X[["control"]])))
-if(!is.null(input_list)&&!(input_list=="")){
-    sampleInfo<-subset(sampleInfo,!(name %in% input_list))
+if(file.exists(yaml_path)){
```

Comment:
in case the yaml file doesn't exist, filtering of input from samples is not done - this is the case in the ATAC-seq workflow. i.e. CSAW.R is used in ChIP-seq (yaml file typically exists, otherwise workflow will fail immediately) and in ATAC-seq (yaml file doesn't exist)

Reply:
Ah, I see. So if the `if` returns `FALSE`, we keep running with different variable values. Okay then!